packages/apify/src/crawlers/basic_crawler.ts

```diff
@@ -678,7 +678,8 @@ export class BasicCrawler {
         }
     }
 
-    protected async _executeHooks(hooks: Hook[], ...args: Parameters<Hook>) {
+    // eslint-disable-next-line space-before-function-paren -- looks like a rule colliding with TS
```

Comment:
yop https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/space-before-function-paren.md

Reply:
should be fixed in master